src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor_Misc.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal partial class SymbolDisplayVisitor
+    {
+        private void AddPreprocessingName(IPreprocessingSymbol preprocessing)
```

Comment:
why do we need this method?  why not just inline it?

Reply:
So as to not clutter the main method that handles symbol types like `IPreprocessingSymbol`, assuming other types could be handled like that in the future. Since it's private, I did not find that much of a concern.

Reply:
i've seen you add a lot of tiny methods that just call out to one more tiny method.  I don't get the purpose of having two methods, one of which just exists to defer to the other.